cli/pkg/providers/aws/aws.go

```diff
@@ -43,7 +43,7 @@ func New(w io.Writer) *provider {
 }
 
 func (p *provider) GetHelmOverrides() []string {
-	return []string{"--set", "global.ngrokEnabled=false"}
+	return []string{"--set", "global.ngrokEnabled=false", "--set", "global.loadbalancer.annotations={\"service.beta.kubernetes.io/aws-load-balancer-type: nlb\"}"}
 }
```

Comment:
Needs to be overwritten at `helm install` time, to avoid re-creation of the `loadbalancer`.